src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,159 @@
+package subway;
+
+
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.jdbc.Sql;
+
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("지하철노선 기능 테스트")
+@Sql("classpath:sql/truncate.sql")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class LineAcceptanceTest {
+
+    /**
+    * When 지하철 노선을 생성하면
+    * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+    */
+
+    @DisplayName("지하철노선 생성")
+    @Test
+    void createLine(){
+        generateStations(new String[]{"왕십리역", "마장역"});
+
+        Map<String, String> params = generateParams("신분당선", "bg-red-600", "1", "2", "10");
+
+        //when
+        ExtractableResponse<Response> response = LineUtils.createLine(params);
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
```

Comment:
```suggestion
        ExtractableResponse<Response> response = LineUtils.createLine(params);
```

response에 대한 검증 코드를 createLine에 넣는다면 더 간결해지지 않을까요?

Reply:
좋은 의견 감사합니다~